mango/dashboards/file_explorer_app.py

```diff
@@ -519,6 +525,24 @@ def _render_file_content(self, path_selected: str, key: str):
                         df = pd.read_excel(excel_file, sheet_name=key_tab)
                         st.dataframe(df, use_container_width=True)
 
+        elif path_selected.endswith(".json"):
+            with st.spinner("Wait for it..."):
+                with open(path_selected, "r") as f:
+                    data = json.load(f)
+                    st.checkbox("As table", value=False, key=f"json_df_{key}")
+                    if st.session_state[f"json_df_{key}"]:
+                        sheets = list(data.keys())
+                        list_of_tabs = st.tabs(sheets)
+                        dict_of_tabs = {}
+                        for i, tab in enumerate(list_of_tabs):
+                            dict_of_tabs[sheets[i]] = tab
```

Comment:
Unless I am missing something, you are just creating the same dict again here.
`dict_of_tabs = {}
                        for i, tab in enumerate(list_of_tabs):
                            dict_of_tabs[sheets[i]] = tab`
is similar to:
`dict_of_tabs = data`

Reply:
I have changed what you said, but I have not put it as you said, because st.tabs in streamlit can't pass the elements one by one.